.github/workflows/cluster_endtoend_vtorc_mysql57.yml

```diff
@@ -115,7 +128,7 @@ jobs:
         set -x
 
         # run the tests however you normally do, then produce a JUnit XML file
-        eatmydata -- go run test.go -docker=false -follow -shard vtorc_8.0 | tee -a output.txt | go-junit-report -set-exit-code > report.xml
+        eatmydata -- go run test.go -docker=false -follow -shard vtorc | tee -a output.txt | go-junit-report -set-exit-code > report.xml
```

Comment:
This needs to be `vtorc_5.7` because we don't want to run the topology tests on 5.7

Reply:
Yes, we would need to change this, or we can do what I suggest https://github.com/vitessio/vitess/pull/11153#discussion_r976281053 and run the topology tests for both versions and remove the vtorc_5.7 entirely from config.json

Reply:
Done